build/karma.conf.js

```diff
@@ -55,7 +55,7 @@ module.exports = function(config) {
 
     logLevel: 'WARN',
 
-    frameworks: ['jasmine', 'browserify'],
+    frameworks: ['jasmine-jquery', 'jasmine', 'browserify'],
```

Comment:
Btw, can you make an implementation comment why 'jasmine-jquery' was added there? Basically say what you said to me.